libs/k8s-client/tests/BaseApi/PodWithLogStreamTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\K8sClient\Tests\PodWithLogStream;
+
+use GuzzleHttp\Handler\CurlHandler;
+use GuzzleHttp\HandlerStack;
+use Keboola\K8sClient\BaseApi\PodWithLogStream;
+use KubernetesRuntime\Client;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\StreamInterface;
+use ReflectionClass;
+
+class PodWithLogStreamTest extends TestCase
+{
+    public function testReadLogReturnsBodyOnSuccess(): void
+    {
+        $namespace = 'default';
+        $podName = 'test-pod';
+        $queries = ['foo' => 'bar'];
+
+        $responseMock = $this->createMock(ResponseInterface::class);
+        $responseMock->method('getStatusCode')->willReturn(200);
+        $responseBody = $this->createMock(StreamInterface::class);
+        $responseMock->method('getBody')->willReturn($responseBody);
```

Comment:
Je potreba mit `Response`  jako mock, kdyz se tam nic neassertuje? `$response = new Response(200)`  by mi prislo, ze bude fungovat uplne stejne.

Reply:
Potřebuji aby to vracelo ten StreamInterface, to mi Response(200) neudělá si myslím

Reply:
![image](https://github.com/user-attachments/assets/bc5ef89b-dc64-41ad-a28f-eefcd8755126)